packages/node-ws/src/events.ts

```diff
@@ -0,0 +1,32 @@
+interface CloseEventInit extends EventInit {
+    code?: number;
+    reason?: string;
+    wasClean?: boolean;
+}
+
+/**
+ * @link https://developer.mozilla.org/en-US/docs/Web/API/CloseEvent
+ */
+export class CloseEvent extends Event {
```

Comment:
```suggestion
export const CloseEvent = globalThis.CloseEvent ?? class extends Event {
```
What do you think about that?
If `CloseEvent` defined, the middleware should use it. This is effective for runtimes where `CloseEvent` is compatible with Node.js is defined.

Reply:
Seems to be good.

If we do so, we may have to have the CI on both Node.js, which has `CloseEvent` and does not.

Reply:
> What do you think about that? If `CloseEvent` defined, the middleware should use it. This is effective for runtimes where `CloseEvent` is compatible with Node.js is defined.

Nice idea, I added your code to the `events.ts`. Thanks for your suggestion!

Reply:
> Currently, `CloseEvent` is not exposed to `globalThis` in Node.js 22.5.0 so I think we don't need to have multiple Node.js versions in the CI for now.

Okay! Let's go with it.